RELEASE_NOTES.md

```diff
@@ -0,0 +1,43 @@
+# v5.1.4 libdrizzle redux (2013-04-25)
+
+# Changelog
+
```

Comment:
BTW, the data above shouldn't be part of the release notes, as the version will be present in the github release (remember this file is just used to collect notes that will be put in the github release page) and the changelog title is redundant :)